lib/diagnostic.js

```diff
@@ -18,6 +18,15 @@
 
 // Answer Starts Here
 //
+// Description: Our app is a browser-based calender for keeping track of things
+// Entity: Task
+// Attributes: date, time, title, description
+// Methods: change time, change date, mark completed
+//
+// Entity: Vacation
+// Attributes: start date, end date, description
+// Methods: auto-respond to email, mark conflicts with work appointments
+//
```

Comment:
Entities generally seem reasonable. I don't know if you can necessarily assume that this app will be integrated with email, though.